packages/blockchain-extension/extension/commands/UserInputUtil.ts

```diff
@@ -1082,6 +1082,17 @@ export class UserInputUtil {
         }
     }
 
+    public static async failedNetworkStart(popupMsg: string, logMsg: string, outputAdapter: VSCodeBlockchainOutputAdapter): Promise<void> {
+
+         const moreInfo: string = 'More Details';
+         outputAdapter.appendLine(LogType.ERROR, logMsg, false);
```

Comment:
You could use `outputAdapter.log` here.
```
VSCodeOutputAdapter.log(type: LogType, popupMessage: string, outputMessage?: string, stackTrace?: string, skipNextLine?: boolean): void
```
So if you just wanted to log something to the output box you could do:
`outputAdapter.log(LogType.ERROR, undefined, logMsg)`